src/app/public/testing/phonefield/phone-field-fixture.ts

```diff
@@ -0,0 +1,147 @@
+import {
+  DebugElement
+} from '@angular/core';
+
+import {
+  ComponentFixture
+} from '@angular/core/testing';
+
+import {
+  By
+} from '@angular/platform-browser';
+
+import {
+  SkyAppTestUtility
+} from '@skyux-sdk/testing';
+
+import {
+  SkyCountryFieldFixture
+} from '@skyux/lookup/testing';
+
+/**
+ * Provides information for and interaction with a SKY UX phone field component.
+ * By using the fixture API, a test insulates itself against updates to the internals
+ * of a component, such as changing its DOM structure.
+ */
+export class SkyPhoneFieldFixture {
+  // this property is lazy loaded and should be accessed via the private countryFixture property
+  private _countryFixture: SkyCountryFieldFixture;
+  private _debugEl: DebugElement;
+
+  /**
+   * The value of the input field for the phone field.
+   */
+  public get inputText(): string {
+    return this.phoneFieldInput.value;
+  }
+
+  constructor(
+    private fixture: ComponentFixture<any>,
+    private skyTestId: string
+  ) {
+    this._debugEl = SkyAppTestUtility
+      .getDebugElementByTestId(fixture, skyTestId, 'sky-phone-field');
+
+    // The country selector needs extra time to initialize.
+    // Consumers shouldn't need to work around this so we do an extra detect here
+    fixture.detectChanges();
+  }
+
+  /**
+   * Sets the value of the input field for the phone field.
+   */
+  public async setInputText(inputText: string): Promise<any> {
```

Comment:
Return type should be `Promise<void>`.

Reply:
Sure I can change that. I was trying to follow what the `whenStable()` method returned, which is `Promise<any>`.

![image](https://user-images.githubusercontent.com/73234923/98586561-7cd58300-228e-11eb-8de8-f9254362f052.png)

Reply:
I would use `void` to avoid confusing the consumers of the fixture so that they're confident nothing of value to them is being returned in the promise (consumers shouldn't "know" that we're returning the result of `fixture.whenStable()`).